pandas/core/groupby/groupby.py

```diff
@@ -1546,15 +1547,16 @@ def backfill(self, limit=None):
 
     @Substitution(name='groupby')
     @Substitution(see_also=_common_see_also)
-    def nth(self, n, dropna=None):
+    def nth(self,
+            n: Union[int, Collection[int]],
```

Comment:
Could arguably go list-like here if we wanted to try and add to _typing but didn't want to stray too far. 

The documentation mentions only supporting lists, though the actual implementation makes accommodations for lists, sets or tuples

Reply:
so generally we would use Sequence then

Reply:
Was thinking that but the isinstance check also includes set, which wouldn't qualify as a Sequence. 

I reverted it to List to keep it simple since Collection isn't available from typing until 3.6 and that is what the documentation shows anyway